src/lib/error/bad-data-error.ts

```diff
@@ -74,10 +74,10 @@ const genericErrorMessage = (
     propertyName: string,
     errorMessage: string = 'is invalid',
 ) => {
-    const input = getProp(requestBody, propertyName);
+    const input = getProp(requestBody, propertyName.split('/'));
 
     const youSent = JSON.stringify(input);
-    const description = `The \`.${propertyName}\` property ${errorMessage}. You sent ${youSent}.`;
+    const description = `The \`${propertyName}\` property ${errorMessage}. You sent ${youSent}.`;
```

Comment:
No other property descriptors start with a full stop, and because the new format uses slashes, I'll remove it for now.